components/Table/Table.vue

```diff
@@ -0,0 +1,108 @@
+<template>
+  <div>
+    <table>
+      <thead>
+        <tr>
+          <th v-for="(head,i) in headers" :key="i">
+            <div :style="align(head.align)">{{ head.text}}</div>
+          </th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr v-for="(row, rowIndex) in items" :key="rowIndex">
+          <td v-for="(header,i) in headers" :key="i">
+            <div :style="align(header.align)">
+              <a v-if="header.link" :href="row[header.value]" target="_blank">
+                <img :src="src(header.icon)" :srcset="srcset(header.icon)" />
+              </a>
+              <div v-else-if="header.collapse" class="collapse">
+                <img :src="src(header.icon)" :srcset="srcset(header.icon)" />
+              </div>
+              <div
+                v-else-if="header.type=='number'"
+              >{{ row[header.value] | number(header.decimalDigit) }}</div>
+              <div
+                v-else-if="header.type=='percentage'"
+              >{{ row[header.value] | number(header.decimalDigit) }}%</div>
+              <div v-else>{{ row[header.value] }}</div>
+            </div>
+          </td>
+        </tr>
+      </tbody>
+    </table>
+  </div>
+</template>
+
+<script>
+export default {
+  name: "mesgTable",
```

Comment:
Let's keep the same than on the other components

```suggestion
  name: "Table",
```

Reply:
Done.